package.json

```diff
@@ -95,5 +95,8 @@
     "atLeast": 100,
     "detail": true,
     "strict": true
+  },
+  "volta": {
```

Comment:
I can remove this, if you want, but https://volta.sh keeps developers using all the same toolkit for each project, which is nice :D

Reply:
I quite like the fact that maintainers are using different versions. Our users also use different Node (and dependency) versions. It catches bugs faster!